testml-lang/testml
# packaging improvements

Using nox test under multiple python versions and verify package imports.

to test, install dependencies with:
`pip install -e .[dev]`

run tests with nox:

```
$nox 
...
nox > Session test_import-3.7 was successful.
nox > Ran multiple sessions:
nox > * test_import-2.7: success
nox > * test_import-3.5: skipped
nox > * test_import-3.7: success
```
python 3.5 is skipped because I don't have it installed.
it should be possible to get travis to test all versions (I am new to nox so I am not sure exactly how, also you already have a travis file and I am not sure how to add python there without breaking things).
